experimenter/experimenter/nimbus_ui_new/templates/nimbus_experiments/edit_overview.html

```diff
@@ -0,0 +1,184 @@
+                Learn more
+              </a>
+            </label>
+          </div>
+          <div class="col-2 text-end">{{ form.risk_brand }}</div>
+        </div>
+        <div class="row mb-3">
+          <div class="col-10">
+            <label for="id_risk_message" class="form-label">
+              Does your experiment include ANY messages? If yes, this requires the
+              <a target="_blank"
+                 href="https://mozilla-hub.atlassian.net/wiki/spaces/FIREFOX/pages/208308555/Message+Consult+Creation">
+                Message Consult
+              </a>
+            </label>
+          </div>
+          <div class="col-2 text-end">{{ form.risk_message }}</div>
+        </div>
+        <div class="row mb-3">
+          <div class="col">
+            <label for="id_application" class="form-label">Application</label>
+            <input class="form-control"
+                   value="{{ experiment.get_application_display }}"
+                   disabled>
+            <p class="form-text">
+              Experiments can only target one Application at a time.
+              <br>
+              Application can not be changed after an experiment is created.
+            </p>
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col">
+            <label for="id_projects" class="form-label">Team Projects</label>
+            {{ form.projects }}
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col">
+            <label for="id_public_description" class="form-label">Public Description</label>
+            {{ form.public_description }}
+            <p class="form-text">This description will be public to users on about:studies</p>
+          </div>
+        </div>
+        <div class="row mb-3">
+          <div class="col-10">
+            <label for="id_risk_revenue" class="form-label">
+              Does this experiment have a risk to negatively impact revenue (e.g. search, Pocket revenue)?
+              <a target="_blank"
+                 href="https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-riskREV">
+                Learn more
+              </a>
+            </label>
+          </div>
+          <div class="col-2 text-end">{{ form.risk_revenue }}</div>
+        </div>
+        <div class="row mb-3">
+          <div class="col-10">
+            <label for="id_risk_partner_related" class="form-label">
+              Does this experiment impact or rely on a partner or outside company (e.g. Google, Amazon) or deliver any encryption or VPN?
+              <a target="_blank"
+                 href="https://mana.mozilla.org/wiki/display/FIREFOX/Pref-Flip+and+Add-On+Experiments#PrefFlipandAddOnExperiments-Isthisstudypartnerrelated?riskPARTNER">
```

Comment:
for links like these, in my current work I am making the constant dict and will use it here, I can replace all that later, just wanted to let you know